extensions-ocp-rhel-9.6.yaml

```diff
@@ -0,0 +1 @@
+extensions-ocp-rhel-9.4.yaml
```

Comment:
if this is going to be a symlink to `extensions-ocp-rhel-9.4.yaml` (which is itself a symlink to `extensions-rhel-9.4.yaml`) then why create `extensions-rhel-9.6.yaml` at all?